Rakefile

```diff
@@ -128,14 +128,18 @@ task :c2m_check_version_on_dir, [:last_checked_b4_date, :storage_root, :profile]
   root = args[:storage_root].to_sym
   storage_dir = "#{Settings.moab.storage_roots[root]}/#{Settings.moab.storage_trunk}"
   last_checked = args[:last_checked_b4_date].to_sym
-
-  if args[:profile] == 'profile'
-    puts "When done, check log/profile_c2m_check_version_on_dir[TIMESTAMP].txt for profiling details"
-    CatalogToMoab.check_version_on_dir_profiled(last_checked, storage_dir)
-  elsif args[:profile].nil?
-    CatalogToMoab.check_version_on_dir(last_checked, storage_dir)
+  begin
+    if args[:profile] == 'profile'
+      puts "When done, check log/profile_c2m_check_version_on_dir[TIMESTAMP].txt for profiling details"
+      CatalogToMoab.check_version_on_dir_profiled(last_checked, storage_dir)
+    elsif args[:profile].nil?
+      CatalogToMoab.check_version_on_dir(last_checked, storage_dir)
+    end
+    puts "#{Time.now.utc.iso8601} Catalog to Moab version check on #{storage_dir} is done."
+  rescue StandardError
```

Comment:
here i think i'd be more specific and `rescue ArgumentError, TypeError`

Reply:
that sounds good, i'll add it. I'm wondering if there are any other Error's I'm leaving out that we don't know about.

Reply:
maybe, but i think better to let them bubble up.  i think you only want to catch stuff you expect and know how to handle.